localization/xliff/enu/constants/localizedConstants.enu.xlf

```diff
@@ -554,6 +554,12 @@
         <trans-unit id="nodeErrorMessage">
             <source xml:lang="en">Parent node was not TreeNodeInfo.</source>
         </trans-unit>
+        <trans-unit id="deleteCredentialError">
+            <source xml:lang="en">Failed to delete credential with id: {0}, Error: {1}</source>
```

Comment:
does localization work with typescript variables like that?

Reply:
yes, need to use Utils.formatString()